src/main/java/edu/escuelaing/ieti/matchmaking/persistence/ActivityRepository.java

```diff
@@ -0,0 +1,20 @@
+package edu.escuelaing.ieti.matchmaking.persistence;
+
+import edu.escuelaing.ieti.matchmaking.exception.MatchmakingException;
+import edu.escuelaing.ieti.matchmaking.model.Activity;
+
+import java.util.List;
+
+public interface ActivityRepository {
+
+    Activity create(Activity activity) throws MatchmakingException;
```

Comment:
Does it make sense to throw a generic Matchmaking exception for every repo method? Should the exception be checked or unchecked?